web/massastation/src/layouts/LayoutStation/LayoutStation.tsx

```diff
@@ -0,0 +1,98 @@
+import { ReactNode, useState } from 'react';
+import { useLocalStorage } from '../../custom/useLocalStorage';
+
+import { useResource } from '../../custom/api';
+import { AccountObject } from '../../models/AccountModel';
+import { useAccountStore } from '../../store/store';
+import { URL } from '../../const/url/url';
+
+import {
+  ThemeMode,
+  StationLogo,
+  Dropdown,
+  Identicon,
+} from '@massalabs/react-ui-kit';
+
+export interface LayoutStationProps {
+  children?: ReactNode;
+  navigator?: Navigator;
+  logo?: ReactNode;
+  onSetTheme?: () => void;
+  activePage: string;
+  customClass?: string;
+  overwrittenTheme?: string;
+}
+
+export function LayoutStation({ ...props }) {
+  const {
+    children,
+    navigator,
+    overwrittenTheme,
+    onSetTheme,
+    activePage,
+    customClass,
+  } = props;
+
+  const [storedTheme] = useLocalStorage<string>(
+    'massa-station-theme',
+    'theme-dark',
+  );
+  const [selectedTheme, setSelectedTheme] = useState(
+    overwrittenTheme || storedTheme || 'theme-dark',
```

Comment:
when do we overwrite the theme ? by curiosity

Reply:
This was already present on the ui-kit. I think it's just to override theme if one page is different by other.